docs/sources/investigate/index.md

```diff
@@ -0,0 +1,196 @@
+---
+description: Investigate trends and spikes to identify issues.
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/investigate/
+keywords:
+  - Explore Traces
+  - Investigate
+title: Investigate trends and spikes
+menuTitle: Investigate trends and spikes
+weight: 600
+---
+
+# Investigate trends and spikes
+
+{{< docs/public-preview product="Explore Traces" >}}
+
+Explore Traces provides powerful tools that help you identify and analyze problems in your applications and services.
+
+Using these steps, you can use the tracing data to investigate issues.
+
+1. Select the primary signal.
+1. Choose the metric you want to use: rates, errors, or duration.
+1. Define filters to refine the view of your data.
+1. Use the structural or trace list to drill down into the issue.
+
+{{< docs/play title="the Grafana Play site" url="https://play.grafana.org/a/grafana-exploretraces-app/explore" >}}
+
+## Change the primary signal type to observe
+
+Tracing data is highly structured and annotated and reflects events that happen in your services.
+You can choose the type of services you want to observe and think about.
+
+You can use the full trace (trace roots), or you can select a more specific type, such as service or database calls.
+
+By default, Explore Traces displays information about full traces. You can change this by using the selector in the Filter bar.
+
+![Selecting a signal type](../images/explore-traces-select-signal.png)
+
+You can use any one of these primary signal types.
+
+Full traces
+: Inspect full journeys of requests across services
+
+Server spans
+: Explore service-specific segments of traces
+
+Consumer spans
+: Analyze how queues are consumed. A consumer span indicates an asynchronous handoff between services. These handoffs are almost always a queue.
+
+Database calls
+: Evaluate performance issues in database interactions
+
+All spans
+: View and analyze raw span data
+
```

Comment:
This area is a UI explanation. The order and list match the UI menu. I'm not sure about changing it around here since we're explaining the UI specifically.